spec/requests/v0/disability_compensation_form_spec.rb

```diff
@@ -330,6 +331,73 @@ def build_optional_xx_dates
     end
   end
 
+  describe 'SavedClaim::DisabilityCompensation::Form526AllClaim save error logging' do
+    let(:form_params) { File.read 'spec/support/disability_compensation_form/all_claims_fe_submission.json' }
+    let(:claim_with_save_error) do
+      claim = SavedClaim::DisabilityCompensation::Form526AllClaim.new
+      errors = ActiveModel::Errors.new(claim)
+      errors.add(:form, 'Mock form validation error')
+      allow(claim).to receive_messages(errors:, save: false)
+      claim
+    end
+    let!(:in_progress_form) { create(:in_progress_form, form_id: FormProfiles::VA526ez::FORM_ID, user_uuid: user.uuid) }
+
+    context 'when the disability_526_track_saved_claim_error Flipper is enabled' do
+      before do
+        Flipper.enable(:disability_526_track_saved_claim_error)
```

Comment:
Please correct Flipper.disable and Flipper.enable with the correct syntax, starting with allow`(Flipper).to receive(:enabled?).with`

Reply:
I think I did this correctly in the last two commits, is this something the Linter could catch or is it trickier to implement a rule for something like this?